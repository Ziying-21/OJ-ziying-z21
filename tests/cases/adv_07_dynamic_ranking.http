###
# Send request:
POST http://127.0.0.1:12345/users HTTP/1.1
content-type: application/json

{"name":"user1"}
###
# Got response:
# HTTP 200
# content-length: 23
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:43 GMT
# 
# {"id":1,"name":"user1"}
###
# Send request:
POST http://127.0.0.1:12345/users HTTP/1.1
content-type: application/json

{"name":"user2"}
###
# Got response:
# HTTP 200
# content-length: 23
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:43 GMT
# 
# {"id":2,"name":"user2"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { std::thread::sleep(std::time::Duration::from_millis(500)); println!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 436
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:43 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:44.312Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { std::thread::sleep(std::time::Duration::from_millis(500)); println!(\"Hello World!\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 483
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:44 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":129441},{"id":1,"info":"","memory":0,"result":"Running","time":793913}],"created_time":"2022-09-09T02:24:44.312Z","id":0,"result":"Accepted","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { std::thread::sleep(std::time::Duration::from_millis(500)); println!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-09T02:24:45.238Z"}
