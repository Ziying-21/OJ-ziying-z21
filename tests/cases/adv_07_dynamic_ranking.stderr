[2022-09-09T02:24:43Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:24:43Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:24:44Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000254
[2022-09-09T02:24:44Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /users HTTP/1.1" 200 23 "-" "-" 0.000757
[2022-09-09T02:24:44Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /users HTTP/1.1" 200 23 "-" "-" 0.000593
[2022-09-09T02:24:44Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 436 "-" "-" 0.000445
[2022-09-09T02:24:45Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 483 "-" "-" 0.000299
[2022-09-09T02:42:17Z INFO  actix_server::server] SIGTERM received; starting graceful shutdown
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
[2022-09-09T02:42:17Z INFO  actix_server::accept] Accept thread stopped
[2022-09-09T02:42:17Z INFO  actix_server::worker] Shutting down idle worker
